torchqdynamics/solvers/mesolve.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+
+from torchqdynamics.solvers._base import FixedStepOption, VariableStepOption
+
+
+@dataclass
+class Rouchon(FixedStepOption):
+    """ TODO: write description """
```

Comment:
A general remark, I suggest we ignore the missing documentation todos for now. We are going to have a lot of uncommented methods here and there, and it's fine. The day we want to comment properly the code base (for example to have an online documentation), we'll spend a day going through the code and adding the missing comments here and there. So I don't think we need to clutter the code with documentation todos everywhere for now. 😉